bp_common/syn/vivado/tcl/vivado_synth.tcl

```diff
@@ -0,0 +1,43 @@
+# Genesys 2 - xc7k325tffg900-2
+# Artix 7   - xc7a200tfbg676-2
+
+set BP_TOP_DIR $::env(BP_TOP_DIR)
+set BP_COMMON_DIR $::env(BP_COMMON_DIR)
+set BP_BE_DIR $::env(BP_BE_DIR)
+set BP_FE_DIR $::env(BP_FE_DIR)
+set BP_ME_DIR $::env(BP_ME_DIR)
+set BASEJUMP_STL_DIR $::env(BASEJUMP_STL_DIR)
+set HARDFLOAT_DIR $::env(HARDFLOAT_DIR)
+
+set f [split [string trim [read [open "flist.vcs" r]]] "\n"]
+set flist [list ]
+set dir_list [list ]
+foreach x $f {
+  # If the item has a file to be added, it starts with $BP_*_DIR
+  if {[string match "\$" [string index $x 0]]} {
```

Comment:
String match # or // and skip those lines instead, default should be file.  That will let wrapper.sv be parsed